.gitlab/internal_image_deploy/internal_image_deploy.yml

```diff
@@ -13,7 +13,7 @@
     - export RELEASE_STAGING="true"
     - export DYNAMIC_BUILD_RENDER_RULES="agent-build-only"
     # Job specific variables
-    - export RELEASE_TAG="${CI_COMMIT_REF_SLUG}${RELEASE_TAG_SUFFIX}"
```

Comment:
as discussed on Slack, let's push both tags first and incrementally migrate to the new one!